src/packages/solid/components/Column/Column.tsx

```diff
@@ -36,11 +41,18 @@ export interface ColumnProps extends IntrinsicNodeProps {
 }
 
 const Column: Component<ColumnProps> = (props: ColumnProps) => {
+  const onUp = handleNavigation('up');
+  const onDown = handleNavigation('down');
+
   return (
-    <SolidColumn
+    <View
       {...props}
-      style={props.style ? [props.style, styles.Container] : styles.Container}
+      onUp={chainFunctions(props.onUp, onUp)}
+      onDown={chainFunctions(props.onDown, onDown)}
+      selected={props.selected || 0}
```

Comment:
so it looks like props.selected will:
1. set an initially selected item
2. override the currently selected item if set during the lifecycle of the component.

however if it's read, it wont provide the currently selected index right? might be nice to support that, or if this is only supposes to be initial then change the prop name